docs-gen/content/tools/vss_documentation.md

```diff
@@ -5,9 +5,9 @@ weight: 3
 ---
 
 The VSS documentation is realized with GitHub Pages. It is generated from
-the markdown files in the ```/documentation``` directory of this repositories.
-The static webpage is generated into the ```/docs``` directory so that it can
-be deployed straight from the repository.
+the markdown files in the ```/docs-gen``` directory of this repository.
+The static webpage is generated after every PR and deployed into a branch 
```

Comment:
"automatically after every PR merged to master"?

Reply:
I will add the change and merge as decided on 08/06/21